testpmd/hooks/roles/mirror-rh-nfv/defaults/main.yml

```diff
@@ -2,4 +2,4 @@
 mrn_index_image: "{{ example_cnf_index_image | default('quay.io/rh-nfv-int/nfv-example-cnf-catalog:v0.2.6') }}"
 mrn_local_registry: "{{ local_registry_host | default('registry.dfwt5g.lab') }}:{{ local_registry_port | default('4443') }}"
 mrn_mirror_index_image: "{{ mrn_index_image | regex_replace('^[^/]+', mrn_local_registry) }}"
-mrn_pull_secret: "{{ dci_pullsecret_file|default('/opt/cache/pull-secret.txt') }}"
+mrn_pull_secret: "{{ partner_creds | default('/opt/cache/pull-secret.txt') }}"
```

Comment:
As default value, shouldn't be use partner_config.json instead of pull-secret.txt for d-o-a-a?

Reply:
Good observation, `/var/lib/dci/...` is a path for our deployment in dallas, the `/opt/cache` is more generic and we know it will be there if we're using doa.

I wonder if we really need a default, maybe that's the right approach, i.e. to not define a default, and only act if partner_creds were provided.  I'll go that route, let me know if you disagree with it

Reply:
Change looks nice, let's test it to confirm it works!